activities/Calligra.activity/index.html

```diff
@@ -5,7 +5,7 @@
 <meta charset="utf-8" />
 <title>Calligra Activity</title>
 <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, viewport-fit=cover"/>
-<link rel="prefetch" type="application/l10n" href="locale.ini" />
+<!-- <link rel="prefetch" type="application/l10n" href="locale.ini" /> -->
```

Comment:
Remove the line, do not comment

Reply:
@llaske, I've made some changes, and now everything is functioning correctly locally. However, I'm having trouble understanding the conflicting file.